tests/blas_like/Multiply.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+   Copyright (c) 2009-2016, Jack Poulson
```

Comment:
I would recommend adding your name as the first author on this file since you introduced it.